kernel/src/process.rs

```diff
@@ -927,6 +943,10 @@ pub enum FaultAction {
 /// This is public for external implementations of `Process`.
 #[derive(Copy, Clone)]
 pub enum Task {
+    /// A task that should not actually be executed. This is used to resume a
+    /// suspended process without invoking any callbacks in userspace (e.g.,
+    /// when YieldFor resolves to a Null Upcall).
+    NullSubscribableUpcall(NullSubscribableUpcall),
```

Comment:
This addition seems not strictly necessary. If we make `fn_ptr` in `FunctionCall` optional, we can re-use that enum option.

Perhaps it could be `FunctionalCallOrReturn`?